File: Java/Hello/src/a26_0827/exam/Product.java
package a26_0827.exam;

public class Product {
  private String name; // 상품명
  private int price; // 상품 가격
  private int stock; // 재고 수량

  public Product(String name, int price, int stock) {
    this.name = name;
    this.price = price;
    this.stock = stock;
  }

  @Override
  public String toString() {
    return "상품명: [" + name + "], 가격: [" + price + "]원, 재고: [" + stock + "]개";
  }

  public int getTotalValue() {

    return price * stock;
  }

  public boolean isAvailable() {
    return stock > 0;
  }

}

File: vscodec/c26_0909/product.cpp
#include <iostream>
#include <string>
using namespace std;

class Product {
public:
    string name;
    int price;

    Product(string n, int p) : name(n), price(p) {}

    void show() {
        cout << name << " : " << price << "원" << endl;
    }
};

int main() {
    Product p("노트북", 1200000);
    p.show();
    return 0;
}